frontend/src/Editor/Components/Checkbox.jsx

```diff
@@ -37,18 +37,19 @@ export const Checkbox = function Checkbox({
   }
 
   return (
-    <div style={{ width, height, display:parsedWidgetVisibility ? '' : 'none' }} onClick={event => {event.stopPropagation(); onComponentClick(id, component)}}>
-      <label className="form-check form-check-inline">
-        <input
-          disabled={parsedDisabledState}
-          className="form-check-input"
-          type="checkbox"
-          onClick={(e) => {
-            toggleValue(e);
-          }}
-        />
-        <span className="form-check-label" style={{color: textColor}}>{label}</span>
+    <div className="row" style={{ width, height, display:parsedWidgetVisibility ? '' : 'none' }} onClick={event => {event.stopPropagation(); onComponentClick(id, component)}}>
```

Comment:
Instead of inline style, you should pass classNames as props.

Reply:
these props and classNames are not related to this PR.